coordinator/migrations/2023-09-29-222654_add_liquidity_requests_table/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE liquidity_requests (
+    id SERIAL PRIMARY KEY,
+    trader_pk TEXT NOT NULL,
+    timestamp TIMESTAMP WITH TIME ZONE NOT NULL NOT NULL,
```

Comment:
Is the second `not null` redundant? 
```suggestion
    timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
```